pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/ModelParser.groovy

```diff
@@ -306,7 +306,25 @@ class ModelParser {
                             break
                         case 'steps':
                             def stepsBlock = matchBlockStatement(s);
-                            stage.branches.addAll(parseStepsBlock(asBlock(stepsBlock.body.code)))
+                            BlockStatement block = asBlock(stepsBlock.body.code)
```

Comment:
I prefer to avoid multi-return methods, which is what I'd ned to get both the branches and `failFast` returned from a method. This code's literally used nowhere but here, so inline is fine, IMO.